import-export-cli/cmd/mg/add.go

```diff
@@ -28,7 +28,7 @@ const (
 	addCmdLongDesc  = `Add new environment and its related endpoints to the config file`
 )
 const addCmdExamples = utils.ProjectName + " " + mgCmdLiteral + " " + addCmdLiteral + " " + envCmdLiteral +
-	" prod --host  https://localhost:9443" +
+	" prod --host https://localhost:9843" +
```

Comment:
are we keeping the flag as host? Shouldn't it be good to have it as adapter?

Reply:
We can change the flag to --adapter. Updating this, and the docs.

Reply:
great. Let's update here too

Reply:
Updating in add.go